com.github.dariasteam.cowsrevenge.json

```diff
@@ -0,0 +1,113 @@
+{
+  "app-id": "com.github.dariasteam.cowsrevenge",
+  "branch": "stable",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "cowsrevenge",
+  "finish-args": [
+     "--socket=x11",
+     "--socket=pulseaudio",
+     "--persist=.godot",
+     "--share=ipc",
+     "--device=dri"
+  ],
+  "build-options" : {
+    "split" : false,
+    "no-debuginfo" : true
+  },
+  "modules": [
+    {
+      "name": "scons",
+      "buildsystem": "simple",
+      "cleanup": [ "*" ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://downloads.sourceforge.net/project/scons/scons/2.5.1/scons-2.5.1.tar.gz",
+          "sha256": "0b25218ae7b46a967db42f2a53721645b3d42874a65f9552ad16ce26d30f51f2"
+        }
+      ],
+      "build-commands": [ "python setup.py install --prefix=/app" ]
+    },
+    "shared-modules/glu/glu-9.0.0.json",
+    {
+      "name": "godot",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/godotengine/godot",
+          "commit": "a524cc92e7cccd012d028d870e4648c59473d04e",
+          "branch": "2.1"
+        }
+      ],
+      "build-commands": [
+          "scons platform=server \\
+            unix_global_settings_path=/app \\
+            prefix=/app \\
+            use_llvm=yes \\
+            -j `nproc` \\
+            use_llvm=yes",
+            "install -D -m755 bin/godot* /app/bin/godot-bin"
+      ]
+    },
+    {
+      "name": "godot-release-template",
+      "buildsystem": "simple",
+      "sources": [
+          {
+          "type": "git",
+          "url": "https://github.com/godotengine/godot",
+          "commit": "a524cc92e7cccd012d028d870e4648c59473d04e",
+          "branch": "2.1"
+        }
+      ],
+      "build-commands": [
+          "scons platform=x11 \\
+            CCFLAGS=-I/app/include \\
+            unix_global_settings_path=/app \\
+            prefix=/app \\
+            -j `nproc` \\
+            target=release \\
+            use_llvm=yes \\
+            tools=no",
+          "install -D -m755 bin/godot.x11.opt.*.llvm /app/templates/linux_x11_`getconf LONG_BIT`_release"
+      ]
+    },
+    {
+      "name": "cowsrevenge",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/Dariasteam/Cows-Revenge",
+          "commit" : "dc4e65763653d0fff72379532685a8c2391ca05b"
+        }
+      ],
+      "buildsystem": "simple",
+      "build-commands": [
+        "mkdir -p /app/cowsrevenge",
+        "mv * /app/cowsrevenge/",
+        "cd /app/cowsrevenge/; \\
+          mkdir -p build; \\
+          ../bin/godot-bin -export 'Linux X11' build/cowsrevenge",
+
+        "mkdir -p /app/bin/",
+        "mkdir -p /app/share/applications/",
+        "mkdir -p /app/share/metainfo/",
+        "mkdir -p /app/share/icons/hicolor/64x64/apps/",
+        "mkdir -p /app/share/icons/hicolor/128x128/apps/",
```

Comment:
You can use `install -D` to install files and make a directory in one call (with -m644 for non-binaries)